### PR TITLE
Add vue section to frameworks

### PR DESCRIPTION
**Pull request by RobPando**

### CODE REVIEW

**Comment by leosoaivan** [comment]
😆

File: frameworks/vue.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Introduction
+
+VueJS was created by Evan You, it's the youngest of the popular frameworks, but it's quickly rising popularity proves that it's worth paying attention to. Vue is known for being one of the easies frontend framework to learn while still having the same scalable power as all the other popular existing frameworks.
```
Comment (I3uckwheat) [inline frameworks/vue.md]: easiest*

File: frameworks/vue.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# Overview
+
+The reason why Vue is so awesome and easy to learn, is because it utilizes what already exists in the web. The Vue way is to use real html and css to build application, you do not need to learn anything new such as JSX for example, although you can integrate it if that is what you want. Vue has a very clean structure for its components, a medium to large Vue application will usually be structured in single file components and the separation of concerns `template` (html) `script` (javascript) and `style` (css) is why the framework is loved, keeping your code organized and clean. Here is an example from the official Vue docs of what a "hello world" single file component would look like:
```
Comment (I3uckwheat) [inline frameworks/vue.md]: 'in' the web or 'on' the web? I'm not sure on the 'right' terminology.

**Comment by I3uckwheat** [comment]
You've convinced me to learn vue in the new future

**Pull request merged**